### ISSUE TITLE
Syntax, EVENTS should only allow VALUE

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#893]
ok,

```
    EVENTS message EXPORTING
      VALUE(p_sub_obj_type) TYPE string.
```

following should give parser error,

```
    EVENTS message EXPORTING
      p_sub_obj_type TYPE string.
```
